tests/integration/conftest.py

```diff
@@ -5,8 +5,8 @@
 
 import pytest
 
-from namespace_container import activate_private_test_namespace, mount, umount
-from utils import BUGTOOL_DOM0_TEMPL, BUGTOOL_OUTPUT_DIR, run
+from .namespace_container import activate_private_test_namespace, mount, umount
```

Comment:
Is this necessary for the actions? Seems like a step in the wrong direction for the python upgrade as relative imports won't be possible

Reply:
These are just local imports within the same Python test framework module.

Implicit relative imports have been deprecated in Python3, and `mypy` warned about them.

This converts these deprecated implicit module-local imports into explicit relative imports, which are of course not deprecated and of course needed and used by all Python3 modules.

So it converts Python2-style module-local imports into Python3-style module-local imports.

Here is of course a reference for anyone else who wants more examples and background:
https://realpython.com/absolute-vs-relative-python-imports/#relative-imports